spec/system/trn_requests_spec.rb

```diff
@@ -308,18 +308,22 @@
     then_i_should_see_the_home_page
   end
 
-  context 'when the use_dqt_api feature is enabled',
-          skip: 'We need to add Redis to the CI and also to update this test to check that an email was sent' do
+  context 'when the use_dqt_api feature is enabled' do
     it 'displays the TRN returned by the DQT API', vcr: true do
       given_the_use_dqt_api_feature_is_enabled
       when_i_have_completed_a_trn_request
       and_i_press_the_submit_button
       then_i_see_a_message_to_check_my_email
+      and_i_receive_an_email_with_the_trn_number
     end
   end
 
   private
 
+  def and_i_receive_an_email_with_the_trn_number
+    expect(ActionMailer::Base.deliveries.last.subject).to eq('Your TRN is 1275362')
```

Comment:
Cool! I saw Apply have some other helpers like `open_email` and `current_email`: https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/spec/system/candidate_interface/signup_and_signin/candidate_clicking_on_expired_magic_link_spec.rb#L48-L49

I think those come from `capybara-email`: https://github.com/DFE-Digital/apply-for-teacher-training/blob/f60a66a3f6e4e5f48d8372567569dec601c3a466/Gemfile#L145

Do you think it's worth adding them?

Non-blocking.

Reply:
Sure, happy to add them.